src/components/ReportActionItem/IOUPreview.js

```diff
@@ -70,6 +70,8 @@ const propTypes = {
 
         /** Does the iouReport have an outstanding IOU? */
         hasOutstandingIOU: PropTypes.bool,
+
+        state: PropTypes.string,
```

Comment:
What's this?

Reply:
Oh I forgot to undo this it's form a previous change